docs/_sources/tutorials/quickstart.rst.txt

```diff
@@ -20,6 +20,10 @@ In a new console window, cd to the NATS test directory in the Linux for Health c
 
    cd connect/src/test/resources/nats
 
+Ensure that the NodeJS dependencies are installed::
+
+   npm install
+
```

Comment:
Thanks for the PR.  To be sure your get proper credit for this, can you please update only docs/source/quickstart.rst?  The .rst docs under source are what the rest of the site is built from.  Then, we'll pull in your commit and rebuild the site.

Reply:
Ah, gotcha, I'll change that. Thanks!